### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Addresses #113 

# Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

Installed locally

## Test Configuration
* Operating system: OS X Big Sur
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by aburrell** [comment]
Woohoo!  All tests failing as expected.  This is needed to merge before the fix for the log test.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+version = "0.0.10"
+description = 'pysat support for Space Weather Indices'
+readme = "README.md"
+requires-python = ">=3.6"
+license = {file = "LICENSE"}
+authors = [
+    {name = "Angeline Burrell", email = "pysat.developers@gmail.com"}
+]
+classifiers = [
+  "Development Status :: 3 - Alpha",
```
Comment (jklenzing) [inline pyproject.toml:16]: Is this still Alpha?

**Review by jklenzing — APPROVED** [review]
Looks good to me. There's an inconsistency about the "alpha" status between the readme and pyproject. That may be intentional and changed in the RC. The only test failure is expected, and a fix is in the queue.

File: pyproject.toml (hunk 1 above)
Comment (aburrell) [reply pyproject.toml:16]: Eh, I made some RC changes throughout.  The next release will go from alpha to beta and I jumped the gun a little bit.

**Pull request merged**